child-process-for-test.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
```

Comment:
This file should be moved into a `fixtures` directory

Reply:
child-process-for-test.js is not a fixture to me. It's rather a test utility. What do you think @sindresorhus?